src/IssueViz.Security/Taint/TaintIssuesSynchronizer.cs

```diff
@@ -101,5 +114,10 @@ private void ClearStore()
         {
             taintStore.Set(Enumerable.Empty<IAnalysisIssueVisualization>());
         }
+
+        private void SetHasTaintIssuesUIContext(bool hasTaintIssues)
+        {
+            vsMonitorSelection?.SetCmdUIContext(contextCookie, hasTaintIssues ? 1 : 0);
```

Comment:
why the `?`? in other places we just assume that we managed to get the service. Can it ever really be null?
https://github.com/SonarSource/sonarlint-visualstudio/blob/e5f23e84787d2bf93dac139c700750b23ffeec25/src/IssueViz/Selection/AnalysisIssueSelectionService.cs#L47

Reply:
It shouldn`t be (and if it is then VS has bigger problems so it won't matter!).

I think this is left over from the prototype as a convenient way to make the existing tests pass without having to supply a valid configured service provider. I'll remove it.